nnstreamer/tensor_trainer/tensor_trainer_nntrainer.cc

```diff
@@ -60,9 +60,10 @@
  *
  * ## Action
  * When a sub-plugin is loaded at runtime,
- * NNTrainerTrain's num_inputs, num_labels, num_training_samples,
- * num_validation_samples, model_save_path and num_epochs is set by
- * getNNStreamerProperties() and each tensor size is set.
+ * Trainer's model_config, num_inputs, num_labels, num_training_samples,
+ * num_validation_samples, model_save_path, model_load_path and num_epochs is
+ set by
```

Comment:
It looks like an unintended line change.

Reply:
Thank you.